lib/src/core/client.dart

```diff
@@ -279,7 +279,18 @@ class Web3Client {
     return _makeRPCCall('eth_sendRawTransaction',
         [bytesToHex(signed, include0x: true, padToEvenLength: true)]);
```

Comment:
Thanks for your contribution. It looks good, but let's change `sendTransaction` to also use the new method:
```suggestion
    return sendRawTransaction(signed);
```